build/scripts/cleanup_perf.ps1

```diff
@@ -17,21 +18,37 @@ try {
 }
 catch {}
 
-# Move all etl files to the a folder for archiving
-echo "creating ToArchive directory"
-mkdir ToArchive
-echo "moving C:/CPC/DataBackup* to ToArchive"
-mv C:/CPC/DataBackup* ToArchive
-ls ToArchive
+try {
+    C:/CPC/cpc.exe /Stop /SkipClean 2>&1 | Out-Null
```

Comment:
Extracted to global.  

Ideally when CPC is nugetified we won't need 60% of this file.